From 99bd608a259c2ef0cafdbb96aa38c8e934c63955 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Mon, 7 Mar 2016 20:33:46 +0100
Subject: [PATCH] iplink: fix help text syntax

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1251186
Upstream Status: iproute2.git commit 5c2ea5b8c001f
Conflicts: Missing commit 6c5ffb9a2c3db ("iplink: cleanup whitespace and
           checkpatch issues"), which introduces worse conflicts.
	   Relevant changes therefore incorporated here.

commit 5c2ea5b8c001fe3e7c85f2c93b9b05bcb035bafa
Author: Phil Sutter <phil@nwl.cc>
Date:   Wed Mar 2 19:19:51 2016 +0100

    iplink: fix help text syntax

    Get rid of extraneous closing brackets and while here, merge the double
    netns parameter.

    Signed-off-by: Phil Sutter <phil@nwl.cc>
---
 ip/iplink.c | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/ip/iplink.c b/ip/iplink.c
index f444bef..787d743 100644
--- a/ip/iplink.c
+++ b/ip/iplink.c
@@ -70,17 +70,16 @@ void iplink_usage(void)
 	fprintf(stderr, "	                  [ address LLADDR ]\n");
 	fprintf(stderr, "	                  [ broadcast LLADDR ]\n");
 	fprintf(stderr, "	                  [ mtu MTU ]\n");
-	fprintf(stderr, "	                  [ netns PID ]\n");
-	fprintf(stderr, "	                  [ netns NAME ]\n");
+	fprintf(stderr, "	                  [ netns { PID | NAME } ]\n");
 	fprintf(stderr, "	                  [ link-netnsid ID ]\n");
 	fprintf(stderr, "			  [ alias NAME ]\n");
 	fprintf(stderr, "	                  [ vf NUM [ mac LLADDR ]\n");
 	fprintf(stderr, "				   [ vlan VLANID [ qos VLAN-QOS ] ]\n");
 
-	fprintf(stderr, "				   [ rate TXRATE ] ] \n");
+	fprintf(stderr, "				   [ rate TXRATE ]\n");
 
-	fprintf(stderr, "				   [ spoofchk { on | off} ] ] \n");
-	fprintf(stderr, "				   [ query_rss { on | off} ] ] \n");
+	fprintf(stderr, "				   [ spoofchk { on | off} ]\n");
+	fprintf(stderr, "				   [ query_rss { on | off} ]\n");
 	fprintf(stderr, "				   [ state { auto | enable | disable} ] ]\n");
 	fprintf(stderr, "			  [ master DEVICE ]\n");
 	fprintf(stderr, "			  [ nomaster ]\n");
-- 
1.8.3.1

